### DISCUSSION TITLE
ipsec tunnelng issue

### DISCUSSION THREAD

**Discussion by bobbywazir** [strongswan/strongswan#1665] [Q&A]
version  - ubuntu 18.04 version 

we hve created a ipsec tunnell with one of our proivider and  tunnel is setup complete but we getting 
generating QUICK_MODE request 983956324 [ HASH SA No KE ID ID ]
sending packet: from 94.130.157.70[500] to 78.46.47.76[500] (444 bytes)
received packet: from 78.46.47.76[500] to 94.130.157.70[500] (428 bytes)
parsed QUICK_MODE response 983956324 [ HASH SA No KE ID ID ]
received netlink error: Network is unreachable (101)
unable to install source route for 192.168.3.100
CHILD_SA site2{4} established with SPIs cace9ce9_i 067f3332_o and TS 192.168.3.100/32 === 192.168.213.1/32
connection 'site2' established successfully

we are not able to ping his 192.168.213.1 - pls suggest somebody who can really help or look into it .
i am also looking for time to time support for the same .

**Comment by tobiasbrunner** [comment]
Please provide the information requested in #196 (in particular the routing tables). Could be an issue with `onlink` routes, in which case you could either try to avoid that via the network config of your system, or install an explicit route for your local LAN manually. Or you disable the automatic route installation by strongSwan (or ignore the error) and install them yourself (including the `onlink` flag), possibly via an updown script.
